_posts/2023-05-17-save-date-ugm2023.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Save the Date - MDAnalysis UGM 2023
+---
+<img
+src="/public/images/UGM2023_Logo.png"
+title="MDAnalysis UGM 2023 Logo" alt="MDAnalysis UGM 2023 Logo"
+style="float: left; " />
+</p>
+
+We are thrilled to announce the first ever MDAnalysis User Group Meeting (UGM)! The MDAnalysis UGM 2023 will take place September 27-29 (Wednesday-Friday) at the [University of Lisbon][uniL] (Portugal). To foster a connection opportunity for interdisciplinary researchers and developers across biomolecular simulations, soft matter and biophysics, this year's event will be held in-person and free to attend for all.
+
+## What to Expect
+Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
+* Keynote talks from experts in the molecular dynamics simulations space
+* Talks related to developing and using MDAnalysis for diverse scientific applications
+* Tutorials diving into new features
+* Direct access to the MDAnalysis core developer team to help shape future development of MDAnalysis
+* Networking opportunities to foster new collaborations
+* Community events, including a hackathon and social outings
+
```

Comment:
mention hackathon

Reply:
Was included under 'Community events', but changed to make it more prominent